### ISSUE TITLE
Plugin stopped working on Chrome on MacOS

### ISSUE THREAD

**Issue by fess89** [berzniz/github_pr_tree#99]
File tree is not displayed on the current Chrome build:
Version 80.0.3987.122 (Official Build) (64-bit) on MacOS Mojave 10.14.6. 
The faulty behaviour started today on multiple machines, so the cause is most likely a Chrome update.

**Comment by AlexITC** [comment]
It fails on ubuntu 18.04 as well, the console displays these errors:
```
TypeError: Cannot read property 'getElementsByClassName' of null
    at l (better_github_prs.js:1)
    at better_github_prs.js:1
    at Array.forEach (<anonymous>)
    at better_github_prs.js:1
    at Array.forEach (<anonymous>)
    at c (better_github_prs.js:1)
    at n.value (better_github_prs.js:31)
    at n.value (better_github_prs.js:31)
    at Ia (better_github_prs.js:22)
    at Ma (better_github_prs.js:22)
vo @ better_github_prs.js:22
better_github_prs.js:22 Uncaught TypeError: Cannot read property 'getElementsByClassName' of null
    at l (better_github_prs.js:1)
    at better_github_prs.js:1
    at Array.forEach (<anonymous>)
    at better_github_prs.js:1
    at Array.forEach (<anonymous>)
    at c (better_github_prs.js:1)
    at n.value (better_github_prs.js:31)
    at n.value (better_github_prs.js:31)
    at Ia (better_github_prs.js:22)
    at Ma (better_github_prs.js:22)
```

**Comment by sarachai** [comment]
> File tree is not displayed on the current Chrome build:
> Version 80.0.3987.122 (Official Build) (64-bit) on MacOS Mojave 10.14.6.
> The faulty behaviour started today on multiple machines, so the cause is most likely a Chrome update.

+1

**Comment by ammmze** [comment]
FWIW I don't think its a chrome update. I haven't restarted chrome in a while, so no update was installed here. I'm guessing github pushed some updates so its probably looking for some element that doesn't exist anymore or was renamed. Github was having some issues this morning which probably means they were doing some updates.

**Comment by Joonpark13** [comment]
Also happening for me on macOS - currently happening on chrome Version 80.0.3987.122 as well, but I also saw it earlier before updating chrome

**Comment by Hacksore** [comment]
Please see #95

**Comment by ammmze** [comment]
Awesome! Thanks 😀 So long story short the fix is already complete, just waiting for the review process in the chrome extension webstore to make it live.

**Comment by berzniz** [comment]
The version with the fix (1.0.26) is out and should be auto updated in your Chrome/Firefox browsers. You can expedite it by going to `chrome://extensions/` and clicking the "Update" button

**Comment by berzniz** [comment]
Thanks for everyone who reported this bug. If you're enjoying this extension, I'd appreciate if you leave a review on the Chrome store so other developers can find the extension as well:

https://chrome.google.com/webstore/detail/better-pull-request-for-g/nfhdjopbhlggibjlimhdbogflgmbiahc
